dataherald/sql_generator/generates_nl_answer.py

```diff
@@ -45,21 +47,27 @@ def execute(self, query_response: Response) -> Response:
             model_name=os.getenv("LLM_MODEL", "gpt-4"),
         )
         database = SQLDatabase.get_sql_engine(database_connection)
-        query_response = create_sql_query_status(
-            database, query_response.sql_query, query_response, top_k=50
-        )
-        system_message_prompt = SystemMessagePromptTemplate.from_template(
-            SYSTEM_TEMPLATE
-        )
-        human_message_prompt = HumanMessagePromptTemplate.from_template(HUMAN_TEMPLATE)
-        chat_prompt = ChatPromptTemplate.from_messages(
-            [system_message_prompt, human_message_prompt]
-        )
-        chain = LLMChain(llm=self.llm, prompt=chat_prompt)
-        nl_resp = chain.run(
-            question=question.question,
-            sql_query=query_response.sql_query,
-            sql_query_result=str(query_response.sql_query_result),
-        )
-        query_response.response = nl_resp
+
+        if not query_response.sql_query_result:
```

Comment:
it is to avoid process again the sql result. We can assume that we have generated this before and only one to use the sql_result to generae a nl_response